### ISSUE TITLE
Keyword arguments of the optimizer for learning rate schedule

### ISSUE THREAD

**Issue by halidziya** [tensorflow/tensorflow#35092]
This statement raises an error if the learning rate is some learning rate schedule class instance. It does not raise any error if specification is without keyword arguments. 

https://github.com/tensorflow/tensorflow/blob/c10aa0338f928dcc3da2c2660d88c80bcdd01278/tensorflow/python/keras/optimizer_v2/optimizer_v2.py#L259

**Comment by oanush** [comment]
@halidziya ,
Can you please provide more information on the issue? Kindly provide share a simple and standalone code to reproduce the same ?Thanks!

**Comment by halidziya** [comment]
Adam(lr=CustomSchedule()) does not work since  lr renamed to learning_rate in tf.keras.optimizers.Adam
